release-notes/release-notes52.md

```diff
@@ -223,6 +229,93 @@ For information on the `toolbar_mode` option, see: [User interface options - `to
 
 This section describes issues that users of {{site.productname}} 5.2 may encounter, as well as possible workarounds for these issues.
 
+**Outline**
+
+* [Translations missing for new user interface options](#translationsmissingfornewuserinterfaceoptions)
+* [Floating toolbar disconnects from the toolbar in inline mode](#floatingtoolbardisconnectsfromthetoolbarininlinemode)
+* [Microsoft Internet Explorer will incorrectly auto focus the editor when TinyMCE is initialized](#microsoftinternetexplorerwillincorrectlyautofocustheeditorwhentinymceisinitialized)
+* [Incorrect positioning of inline dialogs when using sticky toolbars and the bottom toolbar](#incorrectpositioningofinlinedialogswhenusingstickytoolbarsandthebottomtoolbar)
+
+### Translations missing for new user interface options
+
+#### Issue
+
+New user interface features do not have translations to other supported languages at this time. This includes text related to:
+
+- [The new accessibility settings](#newoptiontoassistwithmarkingimagesasdecorativeforaccessibilitypurposes).
+- [The new checklist menu item](#checklist110).
+
+#### Workaround
+
+No known workaround at the time of release.
+
+### Floating toolbar disconnects from the toolbar in inline mode
+
+#### Issue
+
+> **Note**: This issue will be resolved with the release of {{site.productname}} 5.2.1.
+
+If the editor is set to `inline: true`, and `toolbar_drawer` or `toolbar_mode` is set to `'floating'`, such as:
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  inline: true,
+  toolbar_drawer: 'floating'
+});
+```
+
+or
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  inline: true,
+  toolbar_mode: 'floating'
+});
+```
+
+The floating component of the toolbar will disconnect from the primary toolbar when additional content is added to the editor, causing it to resize (such as adding new lines).
```

Comment:
It could just be me but I feel like this reads a bit weirdly. It almost seems like the use of 'it' is referring to the toolbar not the editor.

Reply:
fixed with more specific language.